grama/eval_pnd.py

```diff
@@ -121,21 +120,32 @@ def eval_pnd(model, df_train, df_test, sign, n=int(1e4), seed=None):
 
     ### Create covariance matrices
     X_cov = zeros((X_sig.shape[0], length, length))
-    for i in range(X_sig.shape[0]):
-        X_cov[i, 0, 0] = X_sig[i, 0]
-        X_cov[i, 1, 1] = X_sig[i, 1]
+    for var in range(length):
```

Comment:
`var` in grama code is usually used to specify variables (it's usually a string or iterable of strings); using `var` here as an index is confusing. Use `i` or another index variable instead